examples/deck.gl/i3s/app.js

```diff
@@ -70,33 +70,29 @@ export default class App extends PureComponent {
     });
 
     // Check if a tileset is specified in the query params
-    const {url, token} = this._loadTilesetFromUrl();
-    let dataUrl = url || EXAMPLES[INITIAL_EXAMPLE_NAME].url;
-    if (token) {
-      dataUrl = `${dataUrl}?token=${token}`;
-    }
-
-    this.setState({dataUrl, token});
-    this._onSelectExample(dataUrl, token);
+    const dataUrl = this._loadTilesetFromUrl() || EXAMPLES[INITIAL_EXAMPLE_NAME].url;
+    this._onSelectExample({url: dataUrl});
   }
 
   _loadTilesetFromUrl() {
     const parsedUrl = new URL(window.location.href);
-    return {
-      url: parsedUrl.searchParams.get('url'),
-      token: parsedUrl.searchParams.get('token')
-    };
+    return parsedUrl.searchParams.get('url');
```

Comment:
Suggest renaming this helper method: `_getTilesetUrlFromBrowserUrlParameters()` or something like that, as it doesn't actually load anythng.